web/vtadmin/planetscale-vtadmin-template/README.md

```diff
@@ -0,0 +1,39 @@
+# VTAdmin Web
```

Comment:
This file is only ever used by planetscale to release @planetscale/vtadmin npm package.